### ISSUE TITLE
Web UI 2.2.40 displays a blank page

### ISSUE THREAD

**Issue by nuqleo** [GNS3/gns3-server#2239]
Web UI 2.2.40 displays a blank page in chromium 113
```
2023-06-06 16:08:19 INFO run.py:218 GNS3 server version 2.2.40
2023-06-06 16:08:19 INFO run.py:220 Copyright (c) 2007-2023 GNS3 Technologies Inc.
2023-06-06 16:08:19 INFO run.py:223 Config file /home/user/.config/GNS3/2.2/gns3_server.conf loaded
2023-06-06 16:08:19 INFO run.py:242 Running with Python 3.11.3 and has PID 3704
2023-06-06 16:08:19 INFO run.py:78 Current locale is ru_RU.UTF-8
2023-06-06 16:08:20 INFO web_server.py:318 Starting server on localhost:3080
2023-06-06 16:08:20 INFO __init__.py:69 Load controller configuration file /home/user/.config/GNS3/2.2/gns3_controller.conf
2023-06-06 16:08:20 INFO __init__.py:73 Controller is starting
2023-06-06 16:08:20 INFO __init__.py:308 Installing base configs in '/home/user/Документы/GNS3/configs'
2023-06-06 16:08:20 INFO __init__.py:321 Installing built-in disks in '/home/user/Документы/GNS3/images/QEMU'
2023-06-06 16:08:20 INFO compute.py:63 Create compute local
2023-06-06 16:08:20 INFO compute.py:363 Connecting to compute 'local'
2023-06-06 16:08:20 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:20 +0000] "GET /v2/compute/capabilities HTTP/1.1" 200 553 "-" "Python/3.11 aiohttp/3.8.4"
2023-06-06 16:08:20 INFO notification_handler.py:50 New client has connected to compute WebSocket
2023-06-06 16:08:20 INFO compute.py:430 Connected to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/bundled HTTP/1.1" 200 10059 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/runtime.baa1121a4737aeb68bb7.js HTTP/1.1" 200 10059 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/main.8448c96e4facbe79a613.js HTTP/1.1" 200 10059 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/polyfills.2f91a039d848e57ff02e.js HTTP/1.1" 200 41419 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/styles.f8555f2eecf8cf87f666.css HTTP/1.1" 200 325904 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:08:34 INFO web_log.py:206 ::1 [06/Jun/2023:13:08:34 +0000] "GET /static/web-ui/assets/favicon.ico HTTP/1.1" 200 15342 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
^C2023-06-06 16:08:42 WARNING web_server.py:170 Server has got signal SIGINT, exiting...
2023-06-06 16:08:42 INFO web_server.py:113 Closing 1 websocket connections...
2023-06-06 16:08:42 INFO compute.py:456 Connection closed to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2023-06-06 16:08:43 INFO notification_handler.py:59 Client has disconnected from compute WebSocket
2023-06-06 16:08:43 INFO web_log.py:206 127.0.0.1 [06/Jun/2023:13:08:20 +0000] "GET /v2/compute/notifications/ws HTTP/1.1" 200 0 "-" "Python/3.11 aiohttp/3.8.4"
2023-06-06 16:08:43 INFO __init__.py:145 Controller is stopping

```
Web UI  2.2.39 works fine
```
2023-06-06 16:05:52 INFO run.py:218 GNS3 server version 2.2.39
2023-06-06 16:05:52 INFO run.py:220 Copyright (c) 2007-2023 GNS3 Technologies Inc.
2023-06-06 16:05:52 INFO run.py:223 Config file /home/user/.config/GNS3/2.2/gns3_server.conf loaded
2023-06-06 16:05:52 INFO run.py:242 Running with Python 3.11.3 and has PID 3099
2023-06-06 16:05:52 INFO run.py:78 Current locale is ru_RU.UTF-8
2023-06-06 16:05:53 INFO web_server.py:318 Starting server on localhost:3080
2023-06-06 16:05:53 INFO __init__.py:69 Load controller configuration file /home/user/.config/GNS3/2.2/gns3_controller.conf
2023-06-06 16:05:53 INFO __init__.py:73 Controller is starting
2023-06-06 16:05:53 INFO __init__.py:308 Installing base configs in '/home/user/Документы/GNS3/configs'
2023-06-06 16:05:53 INFO __init__.py:321 Installing built-in disks in '/home/user/Документы/GNS3/images/QEMU'
2023-06-06 16:05:53 INFO compute.py:63 Create compute local
2023-06-06 16:05:53 INFO compute.py:363 Connecting to compute 'local'
2023-06-06 16:05:53 INFO web_log.py:206 ::1 [06/Jun/2023:13:05:53 +0000] "GET /v2/compute/capabilities HTTP/1.1" 200 553 "-" "Python/3.11 aiohttp/3.8.4"
2023-06-06 16:05:53 INFO notification_handler.py:50 New client has connected to compute WebSocket
2023-06-06 16:05:53 INFO compute.py:430 Connected to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/bundled HTTP/1.1" 200 10059 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/runtime.91a209cf21f6fb848205.js HTTP/1.1" 200 3608 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/main.96be36058f5df0ca7e7f.js HTTP/1.1" 200 3400891 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/assets/gns3_icon.svg HTTP/1.1" 200 11424 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/roboto-latin-500.f5b74d7ffcdf85b9dd60.woff2 HTTP/1.1" 200 16114 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/MaterialIcons-Regular.cff684e59ffb052d72cb.woff2 HTTP/1.1" 200 44542 "http://localhost:3080/static/web-ui/styles.f8555f2eecf8cf87f666.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /v2/projects HTTP/1.1" 200 865 "http://localhost:3080/static/web-ui/server/1/projects" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
2023-06-06 16:06:18 INFO web_log.py:206 ::1 [06/Jun/2023:13:06:18 +0000] "GET /static/web-ui/roboto-latin-400.176f8f5bd5f02b3abfcf.woff2 HTTP/1.1" 200 15978 "http://localhost:3080/static/web-ui/bundled" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
^C2023-06-06 16:06:32 WARNING web_server.py:170 Server has got signal SIGINT, exiting...
2023-06-06 16:06:32 INFO web_server.py:113 Closing 1 websocket connections...
2023-06-06 16:06:32 INFO notification_handler.py:59 Client has disconnected from compute WebSocket
2023-06-06 16:06:32 INFO web_log.py:206 127.0.0.1 [06/Jun/2023:13:05:53 +0000] "GET /v2/compute/notifications/ws HTTP/1.1" 200 447 "-" "Python/3.11 aiohttp/3.8.4"
2023-06-06 16:06:32 INFO compute.py:456 Connection closed to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2023-06-06 16:06:32 INFO __init__.py:145 Controller is stopping

```

**Comment by disputin** [comment]
upgrading from 2.2.38 to 2.2.40, same issue in Chrome.  Web console reveals![image](https://github.com/GNS3/gns3-server/assets/1385211/96f1d85c-a3b7-40ad-b830-f8f6cb4c194d)

**Comment by thisisnotcamilo** [comment]
@disputin how do you downgrade from 2.2.40 to 2.2.39 to test? I'm running GNS3 server on Ubuntu 20.04 and it was installed using the docs: https://docs.gns3.com/docs/getting-started/installation/remote-server/

**Comment by grossmj** [comment]
Sorry about the problem. This is fixed in v2.2.40.1
